auth-api/migrations/README

```diff
@@ -1,3 +1,3 @@
 Generic single-database configuration.
 
-1. Add a new migration: /migrations  poetry run alembic revision --autogenerate -m "migration message"
+1. Add new migration: poetry run flask db revision, make sure DEPLOYMENT_ENV is set to 'migration' in your .env
```

Comment:
@Jxio  proper way of doing it consistent with pay (except for the DEPLOYMENT_ENV variable)